pyamg/tests/test_multilevel.py

```diff
@@ -167,3 +167,10 @@ def test_coarse_grid_solver(self):
                 # subsequent calls use cached data
                 x = s(A, b)
                 assert_almost_equal(A*x, b)
+
```

Comment:
@bensworth I'm not sure if this change is a "ghost" or not.   But, I don't think we want it.  I think the test in `test_smoothing.py` for change_matrix is good enough.